modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/ddl/DdlCommandHandler.java

```diff
@@ -195,20 +196,22 @@ private void handle0(DropTableCommand cmd) throws IgniteCheckedException {
 
         security.authorize(cacheName, SecurityPermission.CACHE_DESTROY);
 
-        qryProcessorSupp.get().dynamicTableDrop(cacheName, cmd.tableName(), cmd.ifExists());
+        qryProc.dynamicTableDrop(cacheName, cmd.tableName(), cmd.ifExists());
     }
 
     /** */
     private void handle0(AlterTableAddCommand cmd) throws IgniteCheckedException {
         isDdlOnSchemaSupported(cmd.schemaName());
 
-        GridQueryTypeDescriptor typeDesc = schemaMgr.typeDescriptorForTable(cmd.schemaName(), cmd.tableName());
+        TableDescriptor tblDesc = schemaMgr.table(cmd.schemaName(), cmd.tableName());
 
-        if (typeDesc == null) {
+        if (tblDesc == null) {
             if (!cmd.ifTableExists())
                 throw new SchemaOperationException(SchemaOperationException.CODE_TABLE_NOT_FOUND, cmd.tableName());
         }
         else {
+            GridQueryTypeDescriptor typeDesc = tblDesc.type();
```

Comment:
May be drop 'Grid' prefix for `GridQueryTypeDescriptor` too?

Reply:
There are several `GridQuery` classes(GridQueryTypeDescriptor, GridQueryIndexDescriptor, GridQueryProperty, etc), so if we rename `GridQueryTypeDescriptor` we should also rename other classes with `GridQuery` prefix. I think it not related directly to the current ticket and can be done separately (there are already a lot of changes made by the current PR).